jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java

```diff
@@ -160,12 +160,6 @@ public HttpClient(HttpClientTransport transport)
         addBean(decoderFactories);
     }
 
-    @Override
-    public void dump(Appendable out, String indent) throws IOException
```

Comment:
Why not dumping the listeners anymore? At least knowing how many are registered and what classes they are is a useful troubleshooting info.